libraries/Microsoft.Bot.Builder.Ai.LUIS/Client/BuiltIn.cs

```diff
@@ -228,7 +228,9 @@ public static bool TryParse(string text, out DateTimeResolution resolution)
                 public override int GetHashCode() => throw new NotImplementedException();
             }
 
+#pragma warning disable SA1201 // Elements should appear in the correct order
```

Comment:
Err, why silence this? Why not move the methods up above the nested types? Afraid of setting a precedence that people can just ignore the rules. Either that or we should go ahead and opt the whole codebase out of that rule, but I don't think that's the way to go.